src/media/js/views/tutorial.js

```diff
@@ -3,9 +3,28 @@ define('views/tutorial',
     function(l10n, z, smartButton, SpatialNavigation) {
     var gettext = l10n.gettext;
 
+    var imageLoader = document.createElement('img');
+
+    // Ensure background image is loaded.
+    function loadBackgroundImage($slide, callback) {
+        imageLoader.src = $slide.find('.slide-image')
+                                .css('background-image')
+                                .replace(/url\(['"]?(.*?)['"]?\)/i, '$1');
+
+        imageLoader.onload = function () {
+            $slide.removeClass('invisible');
+
+            callback();
+        };
```

Comment:
It looks weird for this tutorial if the image did not show up, so I would rather redirect the user back to the homepage when `onerror` happened.